src/SQLiteDatabase.ts

```diff
@@ -68,7 +71,7 @@ export async function sqliteInMemoryDatabaseBuilder(
 
   // migrate the schema
   const strategy = new DropCreateDatabaseAlways("sqlite3", knex);
-  await migrate(schema, strategy);
+  await migrateDBUsingSchema(schema, strategy);
```

Comment:
@darahayes if the `migrateDB` method doesn't have problems with new databases, as Enda said than maybe is better to use it

Reply:
@b1zzu I think for now this is the handiest way that ensures the behaviour stays the same. If the `migrateDBUsingSchema` method changes or gets removed I'll make another PR to switch to `migrateDB` 😄